manage.py

```diff
@@ -2,9 +2,26 @@
 import os
 import sys
 
+
+def is_coverage_parallel_mode():
+    parallel_test_mode = 'test' in sys.argv and '--parallel' in sys.argv
```

Comment:
I'd write :

`
argv = sys.argv

if 'test' not in argv or '--parallel' not in argv:
    return False
`

to quit as soon as possible ad avoid a syscall (yes I'm annoying).